scripts/common.js

```diff
@@ -0,0 +1,137 @@
+import {
+  sampleRUM,
+  loadCSS,
+  loadBlock,
+  loadBlocks,
+  loadHeader,
+  buildBlock,
+  decorateBlock,
+} from './lib-franklin.js';
+
+let placeholders = null;
+
+/**
+ * loads a block named 'v2-footer' into footer
+ */
+function loadFooter(footer) {
```

Comment:
Actually I want remove this function ideally, because we should just use the footer we have from franklin and just move all the stuff we have in v2-footer to footer. That's better idea I believe

Reply:
I agree completely!